kubernetes/istio.go

```diff
@@ -482,7 +482,7 @@ func ParseRegistryConfig(config map[string][]byte) (*RegistryConfiguration, erro
 			for _, iItem := range ajRegistry {
 				if mItem, ok := iItem.(map[string]interface{}); ok {
 					kind := mItem["kind"].(string)
-					log.Infof("Kind %s", kind)
+					log.Infof("Kind %s %s", kind, iItem)
```

Comment:
just FYI.. not wrapping dynamic values in log messages will be confusing/hard to read especially if one happens to be an empty string or if they have spaces in them or if you have multiple values and you don't demarcate them. Our usual way to do this is wrap in brackets to make the log message easier to read. Also, I don't think we want Infof here, Debugf would be better so as not to make the logs noisy.
```suggestion
					log.Debugf("Kind [%s] [%s]", kind, iItem)
```